torch_xla/distributed/xla_multiprocessing.py

```diff
@@ -32,6 +32,10 @@ def _is_xla_config():
   return False
 
 
+# TODO: Some usages of this function are to caculate the number of hosts/worlds,
```

Comment:
@will-cromar I made a few changes here to fix the GPU CI failures. Can you take a look at these new changes?

Reply:
Going by the [torchrun documentation](https://pytorch.org/docs/stable/elastic/run.html#environment-variables), doesn't world size mean the number of processes and local world size mean the number of processes on the current host?

Reply:
Does that contradict to what I say? I can improve the TODO to reduce the confusion.

Reply:
Oh yea, I guess I shouldn't use `host/world` here.